src/main/java/uk/ac/ebi/eva/commons/models/converters/data/DBObjectToVariantSourceEntryConverter.java

```diff
@@ -0,0 +1,137 @@
+package uk.ac.ebi.eva.commons.models.converters.data;
+
+import com.mongodb.BasicDBObject;
+import com.mongodb.DBObject;
+import org.opencb.datastore.core.ComplexTypeConverter;
+import org.opencb.opencga.storage.core.variant.VariantStorageManager;
+
+import uk.ac.ebi.eva.commons.models.data.VariantSourceEntry;
+
+import java.io.IOException;
+import java.util.Map;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+/**
+ * @author Cristina Yenyxe Gonzalez Garcia <cyenyxe@ebi.ac.uk>
+ */
+public class DBObjectToVariantSourceEntryConverter implements ComplexTypeConverter<VariantSourceEntry, DBObject> {
+
+    public final static String FILEID_FIELD = "fid";
+
+    public final static String STUDYID_FIELD = "sid";
+
+    public final static String ALTERNATES_FIELD = "alts";
+
+    public final static String ATTRIBUTES_FIELD = "attrs";
+
+    public final static String FORMAT_FIELD = "fm";
+
+    public final static String SAMPLES_FIELD = "samp";
+
+    static final char CHARACTER_TO_REPLACE_DOTS = (char) 163; // <-- £
+
+    private VariantStorageManager.IncludeSrc includeSrc;
+
+    private DBObjectToSamplesConverter samplesConverter;
+
+    /**
+     * Create a converter between VariantSourceEntry and DBObject entities when
+     * there is no need to provide a list of samples or statistics.
+     *
+     * @param includeSrc If true, will include and gzip the "src" attribute in the DBObject
+     */
+    public DBObjectToVariantSourceEntryConverter(VariantStorageManager.IncludeSrc includeSrc) {
+        this.includeSrc = includeSrc;
+        this.samplesConverter = null;
+    }
+
+
+    /**
+     * Create a converter from VariantSourceEntry to DBObject entities. A
+     * samples converter and a statistics converter may be provided in case those
+     * should be processed during the conversion.
+     *
+     * @param includeSrc       If true, will include and gzip the "src" attribute in the DBObject
+     * @param samplesConverter The object used to convert the samples. If null, won't convert
+     */
+    public DBObjectToVariantSourceEntryConverter(VariantStorageManager.IncludeSrc includeSrc,
+                                                 DBObjectToSamplesConverter samplesConverter) {
```

Comment:
includeSrc removed in next commit.

for the samples converter, if we are not going to add other converters, a method is fine.

The reasons I can think of for adding more converters are:
- general changes to the representation while maintaining old structure.
- adding more fields than just genotypes
- extracting the samples subdocs from the current location in the mongo document

I none of them apply, I will move it to the source entry converter